apps/jobs/urls.py

```diff
@@ -3,9 +3,9 @@
 from . import views
 
 urlpatterns = [
-    url(r'challenge/(?P<challenge_id>[0-9]+)/'
```

Comment:
Yes, I just added it for future use. Morover, I will maintain the different PR's for frontend and backend :+1: .
I also added the test cases regarding the view function `change_submission_data_and_visibility`.
I will also add it in different PR.